info.js

```diff
@@ -2,20 +2,18 @@
 
 onmessage = _ => {
 
-    fetch('locales.json')
-        .then(response => {
-            return response.json();
-        })
-        // ITERATE INSIDE
-        .then(json => {
-            const items = json.features;
-            let length = items.length;
-            items.forEach( item => {
-                let lat = item.geometry.coordinates[0];
-                let long = item.geometry.coordinates[1];
-                let fed = item.properties.subcountry;
-                let city = item.properties.city;
-                postMessage([long, lat, fed, city, length]);
-            });
-        });
+  fetch('locales.json')
+    .then(response => {
+      return response.json();
+    })
+    // ITERATE INSIDE
+    .then(json => {
+      const items = json.features;
+      let length = items.length;
+      items.forEach(item => {
+        let [lat, long] = item.geometry.coordinates;
+        let {subcountry, city} = item.properties
+        postMessage([long, lat, subcountry, city, length]);
```

Comment:
```
let [lat, long] = item.geometry.coordinates;
let {subcountry, city} = item.properties
```
is just another way of writing the exact same thing you had. ES6 only.
It won't impact the rest at all.